### PR TITLE
more functionalities (restart, setFrame, ..)

### PR DESCRIPTION
**Pull request by JoEschner**
- added getTextureRegionAtIndex(int i)
- added restartAnimation()
- added setFrame(int f)

To keep the lightweight method for rendering/updating the AnimatedMapTiles, I use a timeOffset to move inside the different animation frames. I would highly appreciate any kind of criticism on this, through I don't think this is the best method, but can't find a different, better one.

### CODE REVIEW

**Comment by MobiDevelop** [comment]
I am hesitant to accept anymore PRs from you. Both of your previous PRs have failed to compile. Please make sure your changes compile before submitting.

File: gdx/src/com/badlogic/gdx/maps/tiled/tiles/AnimatedTiledMapTile.java [hunk 1]
```diff
@@ -74,7 +74,33 @@ public int getCurrentFrameIndex() {
 
 		throw new GdxRuntimeException("Could not determine current animation frame in AnimatedTiledMapTile.  This should never happen.");
 	}
+	
+	public void restartAnimation(){
+		currentTime = lastTiledMapRenderTime % loopDuration;
+		timeOffset -= currentTime;
+	}
 
+	public void setFrame(int f){
```
Comment (MobiDevelop) [inline gdx/src/com/badlogic/gdx/maps/tiled/tiles/AnimatedTiledMapTile.java]: Please use a more meaningful name for this parameter, `frame` or `index` would be better than just `f`.

**Comment by Tom-Ski** [comment]
Please read https://github.com/libgdx/libgdx/wiki/Contributing

**Comment by JoEschner** [comment]
Yes, you're right. I rushed a bit today. Sorry for the inconvenience. Gonna rethink/rewrite this.

**Pull request closed without merging**